test/model.jl

```diff
@@ -417,18 +417,24 @@ end
 function dummy_optimizer_hook(::JuMP.AbstractModel) end
 
 
-function copy_model_style_mode(use_copy_model, caching_mode)
+function copy_model_style_mode(use_copy_model, caching_mode, filter_mode)
     model = Model(caching_mode = caching_mode)
     model.optimize_hook = dummy_optimizer_hook
     data = DummyExtensionData(model)
     model.ext[:dummy] = data
-    @variable(model, x ≥ 0, Bin)
-    @variable(model, y ≤ 1, Int)
+    @variable(model, x[i = 1:2] ≥ 0, Bin)
```

Comment:
Could you explicitly test all three types of possible containers? They're Arrays, DenseAxisArrays, and SparseAxisArrays: https://jump.dev/JuMP.jl/v0.21.5/constraints/#Constraint-containers-1.

Reply:
I just did that in the latest commit. I was expecting that I could just use an argument to `copy_model_style_mode`, but the `container` argument in both `@variable` and `@constraint` is not evaluated (i.e. the variable name is directly used as a container type, and not the contents of the variable).